runtime/src/accounts.rs

```diff
@@ -319,7 +319,13 @@ impl Accounts {
                                         (account, 0)
                                     }
                                 })
-                                .unwrap_or_default()
+                                .unwrap_or_else(|| {
+                                    let mut default_account = AccountSharedData::default();
+                                    if set_exempt_rent_epoch_max {
+                                        default_account.set_rent_epoch(u64::MAX);
```

Comment:
Do we want to add a code comment here explaining why we're doing this?
Something like: 
"All new accounts must be rent-exempt (enforced in Bank::execute_loaded_transaction). Currently, rent collection sets `rent_epoch` to u64::MAX, but initializing the account with this field already set would allow us to skip rent collection for these accounts."

But better.

Reply:
added one